ports/broadcom/common-hal/busio/SPI.c

```diff
@@ -33,28 +33,6 @@ static SPI1_Type *aux_spi[NUM_SPI] = {NULL, SPI1, SPI2};
 static bool never_reset_spi[NUM_SPI];
```

Comment:
I think you  can remove never_reset machinery for SPI now too since there is no bulk reset.

Reply:
Thanks. I forgot this the last time I added finalisers to something as well.